server/rpc.go

```diff
@@ -50,6 +51,88 @@ type Broadcaster interface {
 	Sign([]byte) ([]byte, error)
 }
 
+type BroadcastSessionsManager struct {
+	broadcastSessions []*BroadcastSession
+	broadcastSessMap  map[string]*BroadcastSession
```

Comment:
Naming is a little inconsistent throughout here.

* sessions / sess
* BroadcastSessionManager.broadcastSessions feels a little long and redundant.

FWIW, I prefer brevity in my naming but I'll let you call the scheme as long as it's consistent. Some suggestions: list / table / lock / createSessions